monkey/infection_monkey/network/fingerprinter_manager.py

```diff
@@ -0,0 +1,14 @@
+import logging
```

Comment:
I think we should rename the module - `manager` is a 🧧.
What about: `fingerprinters`, `fingerprinters_collector`, or `fingerprinters_getter`?